CentralityiOS/CentralityiOS/AppDelegate.m

```diff
@@ -7,20 +7,21 @@
 
 #import "AppDelegate.h"
 #import "Parse/Parse.h"
-
-@interface AppDelegate ()
-@end
+NSString *path;
+NSDictionary *dict;
+NSString *applicationId;
+NSString *clientKey;
```

Comment:
Create global variables for static values. For example,
`static const NSString *kApplicationIdKey = @"applicationId";`
and then use it like
`NSString *applicationId = [dict objectForKey: kApplicationIdKey];`